internal/broker/bind_create_test.go

```diff
@@ -272,6 +275,58 @@ func TestCreateBindingEndpoint(t *testing.T) {
 		}`, fixture.PlanId, customExpirationSeconds), t)
 		require.Equal(t, http.StatusBadRequest, response.StatusCode)
 	})
+
+	t.Run("should return 404 for not existing binding", func(t *testing.T) {
+		// given
+		instanceID := "1"
+		bindingID := uuid.New().String()
+		endpoint := fmt.Sprintf("v2/service_instances/%s/service_bindings/%s?accepts_incomplete=false", instanceID, bindingID)
+
+		// when
+		response := CallAPI(httpServer, "GET", endpoint, "", t)
+
+		// then
+		require.Equal(t, http.StatusNotFound, response.StatusCode)
+	})
+
+	t.Run("should return created kubeconfig", func(t *testing.T) {
+		// given
+		instanceID := "1"
+		bindingID := uuid.New().String()
+		endpoint := fmt.Sprintf("v2/service_instances/%s/service_bindings/%s?accepts_incomplete=false", instanceID, bindingID)
+		body := fmt.Sprintf(`
+		{
+			"service_id": "123",
+			"plan_id": "%s",
+			"parameters": {	
+				"service_account": true	
+				}	
+		}`, fixture.PlanId)
+
+		// when
+		response := CallAPI(httpServer, "PUT", endpoint, body, t)
```

Comment:
You use `http.StatusCreated` but do not use `http.MethodPut`. Is there a reason?